boards/arm/mimxrt1050_evk/doc/mimxrt1050_evk.rst

```diff
@@ -141,8 +145,8 @@ board with the on-chip PLL to generate a 600 MHz core clock.
 Serial Port
 ===========
 
-The MIMXRT1050 SoC has eight UARTs. One is configured for the console and the
-remaining are not used.
+The MIMXRT1050 SoC has eight UARTs. One is configured for the console, another
```

Comment:
Maybe explicitly mention which UARTs are configured:

    ``LPUART1`` is configured for the console, ``LPUART3``
    for the Bluetooth Host Controller Interface (BT HCI), and
    the remaining are not used.

Reply:
Thanks @dbkinder. Done.